Kentor.AuthServices/SAML2P/Saml2Response.cs

```diff
@@ -208,6 +209,7 @@ private void ReadAndValidateInResponseTo(XmlElement xml, Saml2Id expectedInRespo
             Saml2Id inResponseTo,
             string relayState,
             Uri audience,
+            bool enforceXmlns,
```

Comment:
This is part of the main library's public API, so this is a breaking change. You have to add another overload that keeps the existing signature to not break existing code.